bin/chandra_repro

```diff
@@ -2137,6 +2137,34 @@ def set_ardlib(params):
     v2("Observation-specific bad pixel file is set in ardlib.par.")
 
 
+def choose_skyfov_method( asolfiles ):
```

Comment:
We (probably me) should look at the flux/repro image/obs stuff, since I think there's code there that runs skyfov. Perhaps it should be consolidated?

Reply:
It looks like very-similar (if not the same) functionality is used in `srcflux` too (looking at #352 )

Reply:
They are similar but a little different.   Small enough snipet of code that adding the conditionals to pick a|b didn't seem worth it to me.